kiwi_lint/db_columns.py

```diff
@@ -0,0 +1,20 @@
+from pylint import interfaces, checkers
+from pylint.checkers import utils
+
+
+class AutoFieldChecker(checkers.BaseChecker):
+    __implements__ = (interfaces.IAstroidChecker, )
+
+    name = 'db-columns-checker'
```

Comment:
Don't mix singular and plural in names. The attribute under interest is `db_column` so everything should be named after it.